NPM fails to authenticate for Lit-Graph on Deploy

When deploying to Fly.io, the Docker Build command tries to do an NPM install. This fails with the following error:

```
#15 [build 5/7] RUN npm install
#15 2.044 npm notice 
#15 2.044 npm notice New minor version of npm available! 9.6.6 -> 9.8.1
#15 2.044 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.8.1>
#15 2.044 npm notice Run `npm install -g npm@9.8.1` to update!
#15 2.044 npm notice 
#15 2.045 npm ERR! code E401
#15 2.046 npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@laughing-man-studios/lit-graph/2.0.1/8cf5c3b5b6b26f26f931e0caac4c7c7d42fdf4cb - authentication token not provided
#15 2.048 
#15 2.048 npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-07-23T16_20_05_664Z-debug-0.log
#15 ERROR: executor failed running [/bin/sh -c npm install]: exit code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -22,7 +22,8 @@ RUN apt-get update -qq && \
 
 # Mount .npmrc file with Auth Key
 RUN --mount=type=secret,id=NPM_RC \
-   cat /run/secrets/NPM_RC > .npmrc
+   (printf "//npm.pkg.github.com/:_authToken=" \
+   && cat /run/secrets/NPM_RC) > .npmrc
 
 # Install node modules
 COPY --link package.json package-lock.json .

--- a/Dockerfile
+++ b/Dockerfile
@@ -26,7 +26,7 @@ RUN --mount=type=secret,id=NPM_RC \
 
 # Install node modules
 COPY --link package.json package-lock.json .
-RUN npm install -g npm@9.6.6
+RUN npm install -g npm@9.8.1
 RUN npm install
 
 # Copy application code

